angular/test/apps/ng16/src/app/app.component.html

```diff
@@ -0,0 +1,3 @@
+<ion-app>
```

Comment:
With ng14 being the minimal supported version and no longer having the `[environmentInjector]` optional syntax, I can do away with the duplicate `AppComponent` for each individual test app and move this to the base app.